build_deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# AppSRE runs this script to build the container and push it to Quay.
+set -exv
+
+IMAGE_NAME="quay.io/cloudservices/cockpit-cloud"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+
+if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" ]]; then
+    echo "QUAY_USER and QUAY_TOKEN must be set"
+    exit 1
+fi
+
+DOCKER_CONF="$PWD/.docker"
+mkdir -p "$DOCKER_CONF"
+docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
```

Comment:
Just to double-check: Really, docker? Is that still running on RHEL 7?

Reply:
That's what everyone else is doing. I don't want to experiment too much while I cannot even produce first deployment.

https://github.com/RedHatInsights/playbook-dispatcher/blob/master/build_deploy.sh
https://github.com/osbuild/image-builder/blob/main/build_deploy.sh

Reply:
Totally fine, just checking. Thanks!